dashboard/config/libraries/PoemBot.interpreted.js

```diff
@@ -0,0 +1,71 @@
+function poemBotDraw() {
+  if (isVisible) {
+    drawPoem();
+  }
+}
+
+function drawPoem() {
+  var outerMargin = 50;
+  var defaultLineHeight = 50;
+  var defaultFontSize = 25;
+  var cursor = outerMargin;
+  fill('black');
+  noStroke();
+  textSize(defaultFontSize);
+  textAlign(CENTER);
+  if (poem.title) {
+    textSize(defaultFontSize * 2);
+    text(poem.title, 200, cursor);
+    textSize(defaultFontSize);
+    cursor += defaultLineHeight;
+  }
+  if (poem.author) {
+    cursor -= defaultLineHeight / 2;
+    textSize(16);
+    text("by: " + poem.author, 200, cursor);
+    textSize(defaultFontSize);
+    cursor += defaultLineHeight;
+  }
+
+  var lineHeight = (400 - cursor) / poem.lines.length;
+  for (var i = 0; i < poem.lines.length; i++) {
+    var line = poem.lines[i];
+    var width = textWidth(line);
+    textSize(Math.min(defaultFontSize, defaultFontSize * (400 - outerMargin) / width));
+    text(line, 200, cursor);
+    textSize(defaultFontSize);
+    cursor += lineHeight;
```

Comment:
Are there any advantages to keeping this value an integer?  IIRC, some rendering paths in the browser end up with a performance and quality hit when rendered into a non-integer "subpixel" location, but I'm not sure if this is one of them.